parsec/core/gui/workspace_sharing_widget.py

```diff
@@ -132,7 +145,7 @@ def __init__(self, user_info, is_current_user, current_user_role, role):
             # Not enabled, no point in filling all the roles since they cannot be changed anyway
             self.combo_role.addItem(get_role_translation(role))
         else:
-            for role, index in _ROLES_TO_INDEX.items():
+            for role, index in _ROLES_TO_INDEX.items():  # type: ignore[assignment]
```

Comment:
This ignore could be avoid. The issue is that `role` variable is already used. If you rename the variable `role`to `_role`in the for loop it shall solve the problem